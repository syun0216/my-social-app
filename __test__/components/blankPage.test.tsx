import React from 'react';
import renderer from 'react-test-renderer';
import { Text } from 'react-native';

import BlankPage from '../../components/blankPage';

const text = 'Hello world';
const testRender = renderer.create(<BlankPage text={text} />);
const testInstance = testRender.root;

test('renders correctly', () => {
  const tree = testRender.toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders text', () => {
  const textItems = testInstance.findAllByType(Text);
  expect(textItems.length).toBe(1);
  expect(textItems[0].props.children).toEqual(text);
});
