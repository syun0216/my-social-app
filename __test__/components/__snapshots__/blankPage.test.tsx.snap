// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#FAFAFA",
        "flex": 1,
        "justifyContent": "center",
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "marginBottom": 25.78125,
      }
    }
  />
  <Text
    allowFontScaling={false}
    style={
      Object {
        "color": "#BABABA",
        "fontFamily": "SourceSansPro-Regular",
        "fontSize": 33,
      }
    }
  >
    Hello world
  </Text>
</View>
`;
